website/docs/reference/widgets/list.md

```diff
@@ -1,30 +1,24 @@
 # List
 
-The List widget provides a way to iterate over a structured dataset(array of objects) and display the data in repeating sections without writing any code. Each list item can contain other widgets to display data or capture user input.
-
+The List widget provides a way to iterate over a structured dataset (array of objects) and display the data in repeating sections without writing any code. Each list item can contain other widgets to display data or capture user input.
 
 <VideoEmbed host="youtube" videoId="0ePiZlWmp7Q" title="How to use List Widget" caption="How to use List Widget"/>
 
-:::info
-Although you can add many other widgets inside the List widget, currently, Appsmith fully supports only the Text and Image widgets inside a List.
-:::
-
-
 ##  List components
 
-Typically, a List widget is a collection of other widgets you can embed to display data or capture user input. Once you drag a List widget on the canvas, it has embedded Image and Text widgets by default. You can add more widgets per your requirement on the first list item container. The widgets get replicated on the other list items automatically when a dataset is connected to the List. 
+Typically, a List widget is a collection of other widgets you can embed to display data or capture user input. Once you drag a List widget on the canvas, it has embedded Image and Text widgets by default. You can add more widgets per your requirement on the first list item container. When you connect the dataset to the List, the widgets in the subsequent list items get updated with values automatically. 
```

Comment:
Can we also add that the list widget displays items vertically?

Reply:
updated this